services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/TopicCleanupService.java

```diff
@@ -235,49 +238,46 @@ Admin getAdmin() {
    * If version topic deletion takes more than certain time it refreshes the entire topic list and start deleting from RT topics again.
     */
   void cleanupVeniceTopics() {
+    // todo - instead of adding all topics into the same queue we must simply use separate queues for RT and nonRT
```

Comment:
nite, todo -> "TODO" so that IDE can highlight it.

Reply:
in my IDE, small case todo also highlights